### PR TITLE
Update glusterfs persistent storage example to use NGNIX rather than busybox

### PR DESCRIPTION
**Pull request by screeley44**
Updated the existing storage example for gluster and ngnix (replacing busybox references) and revalidated that the instructions worked

### CODE REVIEW

**Comment by screeley44** [comment]
@adellape - PTAL and merge if acceptable, thanks!

File: install_config/storage_examples/gluster_example.adoc [hunk 1]
```diff
@@ -269,26 +322,30 @@ metadata:
...
       claimName: gluster-claim      <6>
 ----
 <1> The name of this pod as displayed by `oc get pod`.
-<2> The image run by this pod. In this case, we are telling *busybox* to sleep.
+<2> The image run by this pod. In this case, we are using standard ngnix image.
 <3> The name of the volume. This name must be the same in both the
 `*containers*` and `*volumes*` sections.
 <4> The mount path as seen in the container.
-<5> The group ID to be assigned to the container.
+<5> The SupplementalGroup ID (Linyux Groups) to be assigned at the pod level
```
Comment (adellape) [inline install_config/storage_examples/gluster_example.adoc]: Linyux typo?

**Comment by adellape** [comment]
@screeley44 One comment, otherwise LGTM. The remaining style edits can be done in a follow-up PR by the docs team. Thanks.

**Comment by screeley44** [comment]
@adellape - fixed the typo - thanks for the quick review!

File: install_config/storage_examples/gluster_example.adoc (hunk 1 above)
Comment (screeley44) [inline install_config/storage_examples/gluster_example.adoc]: fixed

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/storage_examples/gluster_example.adoc#install-config-storage-examples-gluster-example[Persistent Storage Examples -> Complete Example Using GlusterFS]
|Updated the GlusterFS persistent storage example to use NGNIX instead of busybox.
%

**Pull request merged**